lib/src/ui.dart

```diff
@@ -193,6 +207,19 @@ class _CropState extends State<Crop> with TickerProviderStateMixin {
     _endOffset = widget.controller._offset;
 
     setState(() {});
+    updateOnChanged();
+
+  }
+
+  Future updateOnChanged() async{
```

Comment:
I suggest the name `_handleOnChanged`

It doesn't need to be `async` and there's no need to return a `Future`.

Reply:
Removing the Future & async.
But important to note that, it will cause some jank in the example because of the `print` statement.

Shall we also add a new private `async` method   `_handleCropChanged`  to be passed as `onChanged` callback in the example to call  the `print` statement?
I believe, ideally, people would be doing something async there like showing it in the ui or updating some ruler, etc.

What do you think?